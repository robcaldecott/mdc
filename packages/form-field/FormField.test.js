import React from "react";
import { FormField } from ".";
import renderer from "react-test-renderer";

it("renders", () => {
  expect(
    renderer.create(
      <FormField>
        <input type="checkbox" id="checkbox" />
        <label htmlFor="checkbox">Label</label>
      </FormField>
    )
  ).toMatchSnapshot();
});

it("renders alignEnd", () => {
  expect(
    renderer.create(
      <FormField alignEnd>
        <input type="checkbox" id="checkbox" />
        <label htmlFor="checkbox">Label</label>
      </FormField>
    )
  ).toMatchSnapshot();
});
