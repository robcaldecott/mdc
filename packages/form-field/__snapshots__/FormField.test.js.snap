// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div
  className="mdc-form-field"
>
  <input
    id="checkbox"
    type="checkbox"
  />
  <label
    htmlFor="checkbox"
  >
    Label
  </label>
</div>
`;

exports[`renders alignEnd 1`] = `
<div
  className="mdc-form-field mdc-form-field--align-end"
>
  <input
    id="checkbox"
    type="checkbox"
  />
  <label
    htmlFor="checkbox"
  >
    Label
  </label>
</div>
`;
